doc/nrf/releases_and_maturity/releases/release-notes-2.6.1.rst

```diff
@@ -0,0 +1,115 @@
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.6.1`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.6.1.
+See the :ref:`installation` section for more information about supported operating systems and toolchain.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.6.1_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.6.1_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.6.1`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_261_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+
+Matter
+------
+
+* Updated:
+
+  * The default Message Reliability Protocol (MRP) retry intervals for Thread devices to two seconds to reduce the number of spurious retransmissions in Thread networks.
+  * Increased the number of available packet buffers in the Matter stack to avoid packet allocation issues.
+
+Thread
+------
+
+* Fixed ``otPlatCryptoPbkdf2GenerateKey`` API implementation to allow a fallback to legacy MbedTLS implementation instead of returning ``OT_ERROR_NOT_CAPABLE``.
+
+Applications
+============
+
+Matter Bridge
+-------------
+
+* Added:
+
+  * The :kconfig:option:`CONFIG_BRIDGE_BT_MAX_SCANNED_DEVICES` kconfig option to set the maximum number of scanned Bluetooth LE devices.
+  * The :kconfig:option:`CONFIG_BRIDGE_BT_SCAN_TIMEOUT_MS` kconfig option to set the scan timeout.
+
+* Fixed an issue where the recovery mechanism for Bluetooth LE connections might not work correctly.
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``23cf38934c0f68861e403b22bc3dd0ce6efbfa39``.
+
+For a complete list of |NCS| specific commits and cherry-picked commits since v2.6.0, run the following command:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^v3.5.99-ncs1
```

Comment:
@anangl : Can you please confirm if the Zephyr updates are correct?

Reply:
They are correct.
I think we should mention here the QSPI driver fix, see https://github.com/nrfconnect/sdk-nrf/pull/14899#discussion_r1574541243.

Reply:
Done. Please check.

Reply:
Looks good, thanks.